pod/perl.pod

```diff
@@ -392,9 +392,10 @@ see L<perlvar> for more information.
 Using the C<use strict> pragma ensures that all variables are properly
 declared and prevents other misuses of legacy Perl features.
 
-The C<use warnings> pragma produces some lovely diagnostics. One can
-also use the B<-w> flag, but its use is normally discouraged, because
-it gets applied to all executed Perl code, including that not under
+The C<use warnings> pragma produces some lovely diagnostics.
+These are enabled by default within the scope of C<use v5.35> (or higher).
```

Comment:
If this is added here, it would also need to mention strict is enabled by `use v5.12` or higher in the paragraph above.

Reply:
Done, thanks.